static/templates/tab_bar.handlebars

```diff
@@ -7,7 +7,7 @@
         {{! Most tabs are links, but some are not since we don't have a narrow for them (e.g. Search) }}
         {{#if hash}}
             {{#if home}}
-            <a href="{{hash}}"><i class="icon-vector-home"></i></a>
+            <a href="{{hash}}" title="{{t 'Home' }} (Esc, Ctrl + [)"><i class="icon-vector-home"></i></a>
```

Comment:
I'd just put one of the two, probably `(Esc)`.
I'm not sure I how I feel about title text on the left sidebar in general, but I guess this one and Private Messages are fairly important.